src/GraphQLParser/AST/ASTNodeKind.cs

```diff
@@ -0,0 +1,77 @@
+namespace GraphQLParser.AST
+{
+    /// <summary>
+    /// The kind of nodes in the GraphQL document AST (Abstract Syntax Tree).
+    /// </summary>
+    public enum ASTNodeKind
+    {
+        /// <summary>
+        /// Named thing inside GraphQL document: operations, fields, arguments, types, directives, fragments, and variables.
+        /// <br/>
+        /// All names must follow the same grammatical form: [_A-Za-z][_0-9A-Za-z]*
+        /// </summary>
+        Name,
+        Document,
+        OperationDefinition,
+        VariableDefinition,
+        Variable,
+        SelectionSet,
+        Field,
+        Argument,
+        FragmentSpread,
+        InlineFragment,
+        FragmentDefinition,
```

Comment:
Applying a comment to `Name` does not automatically apply it to the other enumeration entries.

Reply:
(does it?)

Reply:
Of course I added comment above only for `Name`.